Sample/Vungle Sample App/FirstViewController.m

```diff
@@ -388,7 +418,7 @@ - (IBAction)showAdForPlacement01 {
 
 - (IBAction)showAdForPlacement02 {
     // Play a Vungle ad (with options). Dictionary to set custom ad options.
-    NSDictionary *options = @{VunglePlayAdOptionKeyOrientations: @(UIInterfaceOrientationMaskLandscape)};
+    NSDictionary *options = @{VunglePlayAdOptionKeyOrientations: @(UIInterfaceOrientationMaskAll),VunglePlayAdOptionKeyOrdinal: @20031023};
```

Comment:
Why are you setting ordinal here as well? Also - why are you using the same value? If ordinal needs to be properly tested, it should be in a var that is incremented after each use.